tbx/core/blocks.py

```diff
@@ -171,7 +195,7 @@ class StoryBlock(StreamBlock):
         label="Wide image",
         template="patterns/molecules/streamfield/blocks/wide_image_block.html",
     )
-    pullquote = PullQuoteBlock(
+    quote = PullQuoteBlock(
```

Comment:
**Issue**: renaming `pullquote` to `quote` _may_ result in a loss of data for existing pull quotes. This is likely to be the case if, when running `./manage.py makemigrations`, you didn't get a prompt to confirm whether you renamed `pullquote` to `quote`.

**suggestion**: maintain the same name and add a `label="Quote"` argument to `PullQuoteBlock`, similar to what was done for `raw_html`.